tests/integration/test_relative_links.py

```diff
@@ -25,9 +25,11 @@ def test_relative_links(build, output_dir):
     with io.open(test_path, "rb") as inf:
         data = inf.read()
 
+    html = lxml.html.fromstring(data)
+
     assert not any(
         url.startswith("..")
-        for _, _, url, _ in lxml.html.iterlinks(data)
+        for _, _, url, _ in html.iterlinks()
```

Comment:
Personally, I think the following denser version is more readable:

```
assert not any(
    url.startswith("..")
    for _, _, url, _ in lxml.html.fromstring(data).iterlinks()
```

That's somewhat of a matter of taste. If you want to leave that extra variable, fine, but then I suggest to delete the empty line (current 29) as it's only needed in the assert.

Reply:
Changed.